### ISSUE TITLE
improve installation doc

### ISSUE THREAD

**Issue by shinyano** [Quansight-Labs/free-threaded-compatibility#111]
In `docs/installing_cpython.md` windows section, the following script:
```powershell
$url = 'https://www.python.org/ftp/python/3.13.0/python-3.13.0-amd64.exe'
Invoke-WebRequest -Uri $url -OutFile 'python.exe'
python.exe /quiet Include_freethreaded=1
```

should be 
```powershell
$url = 'https://www.python.org/ftp/python/3.13.0/python-3.13.0-amd64.exe'
Invoke-WebRequest -Uri $url -OutFile 'python.exe'
.\python.exe /quiet Include_freethreaded=1                           # <- add .\ before python.exe
```

to execute the downloaded installer, because Windows PowerShell does not load commands from the current location by default.

Directly typing `python.exe` may even execute the Python version that was previously installed and added to PATH by the user.

To be more clear, we can change the name of the installer downloaded:
```powershell
$url = 'https://www.python.org/ftp/python/3.13.0/python-3.13.0-amd64.exe'
Invoke-WebRequest -Uri $url -OutFile 'python-3.13.0-amd64.exe'
.\python-3.13.0-amd64.exe /quiet Include_freethreaded=1
```

**Comment by ngoldbaum** [comment]
Makes sense to me, would you mind creating a PR with the change?

**Comment by shinyano** [comment]
Sure! I'd glad to.

**Comment by lysnikolaou** [comment]
Fixed in #112!
